gallery/src/themeClassManager.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { includes } from 'ramda';
+
+const validThemeOverrides = [null, 'light', 'dark'] as const;
+type ThemeOverride = (typeof validThemeOverrides)[number];
+
+export let themingEnabled: boolean = localStorage.getItem('theming-enabled') === 'true',
+    themeOverride: ThemeOverride = getThemeOverride();
+
+function getThemeOverride(): ThemeOverride {
```

Comment:
This is a super small nitpick, I wonder if we can reorder these functions so they don't seem arbitrarily ordered: Maybe something like 🤔 :

- getThemeOverride
- setThemingEnabled
- setThemingOverride
- updateHtmlClasses

or

- setThemingEnabled
- setThemingOverride
- getThemeOverride
- updateHtmlClasses

Let me know what you think!